charts/rollups-node/ci/test-values.yaml.tpl

```diff
@@ -23,6 +23,7 @@ validator:
     CARTESI_FEATURE_READER_MODE: "true"
     CARTESI_FEATURE_DISABLE_MACHINE_HASH_CHECK: "true"
     CARTESI_SNAPSHOT_DIR: "/usr/share/cartesi/snapshot"
+    CARTESI_AUTH_KIND: "mnemonic_file"
```

Comment:
Don't we need to update the image name ? (`cartesi/dapp:echo-python-0.16.0-server`)

Reply:
The CI is passing, since we don't test inputs processing neither do claims.

Since the current CI just tests the deployment, I'm ok with this.

We could improve the CI, and we may need to maintain our own application for this instead of depending on echo-python that used to be built on cartesi/rollups-examples repository and this repository is archived.

It's debatable if we need to have this kind of end-to-end tests on the scope of this chart, since it's already being worked on at the original repository cartesi/rollups-noce.